### DISCUSSION TITLE
CFAST – Consolidated Model of Fire Growth and Smoke Transport (Version 6) Software Development and Model Evaluation Guide

### DISCUSSION THREAD

**Discussion by FireStudent321** [firemodels/cfast#2086] [Q&A]
Hi All,

I'm just wondering where I could find the raw data for NBS Multi-Compartment Test Series. I think these are plotted in graphs in CFAST Handbook Guide (section B.6, page 162-165). Am I right to think that the graphs show the average hot gas layer temperature vs. time for experiments? However, I couldn't find exactly where the raw data are that were used to plot the graphs. 

CFAST Handbook Guide Link: https://nvlpubs.nist.gov/nistpubs/specialpublications/nist.sp.1086r1.pdf

I'm aware that the experimental set-up is taken from "An Experimental Data Set for the Accuracy Assessment of Room Fire Models" report. However, I've noticed that in the "An Experimental Data Set for the Accuracy Assessment of Room Fire Models" report, there are no results for each of the tree. But in CFAST Handbook Guide there are but not sure where it came from. Also, I don't know where the raw data are for each of the thermocouples are. I'm hoping someone would be able to point me in the right direction.

Thanks.

**Comment by drjfloyd** [comment]
Validaton data is archived here:

https://github.com/firemodels/out

**Reply by FireStudent321** [reply]
Hi Dr. Floyd,

Thanks for the reply. Is the output from the FDS model? I was hoping for the experimental raw data. I couldn't find the raw data from "An Experimental Data Set for the Accuracy Assessment of Room Fire Models" report and "An Experimental Data Set for the Accuracy Assessment of Room Fire Models" report.

**Comment by drjfloyd** [comment]
sorry posted the wrong link. I meant to post exp instead of out.

**Reply by FireStudent321** [reply]
No problem. What link can I find the experimental rata data on?

**Comment by gforney** [comment]
https://github.com/firemodels/exp

On Sun, Feb 25, 2024 at 9:11 PM FireStudent321 ***@***.***>
wrote:

> No problem. What link can I find the experimental rata data on?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/firemodels/cfast/discussions/2086#discussioncomment-8586830>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC6UCRUWAKVQ4D4M4W3FIS3YVPVMPAVCNFSM6AAAAABDYQABRWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKOBWHAZTA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Glenn Forney

**Reply by FireStudent321** [reply]
Thanks for the reply, Dr. Forney. 

I just have a few questions to ask about NBS_MV100A.csv:
- I was just wondering what the -10, -20, -30, -40, -50, -60 meant under the heading "time";
- Also, I was wondering if the T#1 6, T#1 14, T#1 26, T#1 38, T#1 50, T#1 62, T#1 74, T#1 80 and T#1 84 corresponds to the 9  
   thermocouples (of tree 1) at 0.15, 0.36, 0.66, 0.97, 1.27, 1.57, 1.88, 2.03 and 2.15m from floor, respectively.

**Comment by drjfloyd** [comment]
Time is the time in s.  The data is 60 s of background data followed by fire ignition at 0 s.

6 in. = 15 cm
14 in. = 36 cm
etc.

**Reply by FireStudent321** [reply]
Thanks, Dr. Floyd.

**Comment by FireStudent321** [comment]
Hi,

Is there a particular reason why the report states that the ambient temperature is 22 degree celsius for MV100Z when the thermocouples read different temperatures at 0 seconds as per the link? The air temperature even reached 49 degree celsius in the fire room (tree 1) at 0 seconds. Where did they get the 22 degree celsius from? Have they taken the lowest air temperature at 0 seconds? 

Link: https://github.com/firemodels/exp/blob/master/NBS_Multi-Room/NBS_MV100Z.csv

I'm just trying to understand why the starting air temperature of the experiment is higher than the starting air temperature of FDS.

![image001-2](https://github.com/firemodels/cfast/assets/161180346/4de16ab3-db08-4715-b2ef-44ff824fd6f9)

Thanks.

**Reply by drjfloyd** [reply]
These tests were a set of compartments built inside of a larger room. I suspect this it that the larger room temperature was 22 C and multiple tests were run in the same day. Most of the TCs are within a few degrees of 22 C. The exception is the upper TCs in the fire room.

**Reply by FireStudent321** [reply]
Thanks, drjfloyd.

I know that this may seem like a general question and not related to CFAST at all. But is there a range or an approximate value for the entrainment coefficient of a wall plume or plume really close to a wall (due to the proximity of a burner to a wall)? I think the entrainment coefficient for a top hat profiled plume is roughly 0.15 (plume far away from walls).

**Reply by drjfloyd** [reply]
See NUREG 2178 Volume 2 Chapter 6 and NIST Technical Note 1984. Both conclude that wall plumes can be treated as open plumes in terms of the plume temperature.

**Reply by FireStudent321** [reply]
Thanks for the reply. It's just that I'm trying to use the empirical equation b=(6/5)(alpha)z for a wall plume to calculate the wall plume's size at the ceiling. b stands for radius, alpha stands for entrainment coefficient and z is for plume height above fire source.

**Comment by FireStudent321** [comment]
Hi Dr. Floyd,

I was just wondering what the T#1 or T#8 means in the link below. 

Link: https://github.com/firemodels/exp/blob/master/NBS_Multi-Room/NBS_MV100Z.csv. 

I am not sure if it corresponds correctly to the number assigned the thermocouple. For example, Tree 8 in the third room does not have a thermocouple that is 1.93m above ground. Also, in the report it says that there are 10 thermocouples in tree 8, but in the link there are only 8 thermocouple results. I was just wondering where the remaining results are. I think this is the same for the other trees as well.

**Reply by rpeacoc** [reply]
These are just identifiers for each measurement in the test. See https://www.nist.gov/publications/experimental-data-set-accuracy-assessment-room-fire-models for the report this data set comes from.

**Reply by FireStudent321** [reply]
Hi rpeacoc, 

Thanks for the reply. 

Does T#8 correspond to Tree 8 and T#1 correspond to Tree 1?

I was hoping to find the raw data, which has the results for each thermocouple on each tree. I think the raw data may have been converted to upper layer or lower layer temperatures in the report.

**Reply by FireStudent321** [reply]
Is the ambient temperature taken outside the set of compartments? Was the experiment conducted inside of a larger room?

**Reply by drjfloyd** [reply]
The exp folder contains 7 files for each test. One file (with no Treex_layer) in the name) is the raw data. The other files are the layer processed data.

**Reply by FireStudent321** [reply]
Thanks for the reply, drjfloyd. 

I'm guessing you are referring to the link below. 
Link: https://github.com/firemodels/exp/blob/master/NBS_Multi-Room/NBS_MV100Z.csv

Tree 8 in the third room does not have a thermocouple that is 1.93m above ground. Also, in the report it says that there are 10 thermocouples in tree 8, but in the link there are only 8 thermocouple results. I was just wondering where the remaining results are.

**Reply by rpeacoc** [reply]
Realize that these tests were conducted nearly 40 years ago. The data we used for the validation of the NIST fires models have been checked back to any original records that exist, but additional data may have been recorded at the time that have not been used in the validation efforts. Details of some of these data are, unfortunately, lost to time.

**Reply by FireStudent321** [reply]
Thanks for the reply, rpeacoc. I was just wondering if T#8 represents Tree 8 and T#1 represents tree 1 in the report.
